diff --git a/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js b/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js
index 8659501..5ccbce7 100644
--- a/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js
+++ b/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js
@@ -1495,6 +1495,7 @@ function (_React$Component) {
 
     var itemSeparator = ItemSeparatorComponent && React.createElement(ItemSeparatorComponent, this.state.separatorProps);
     var cellStyle = inversionStyle ? horizontal ? [styles.rowReverse, inversionStyle] : [styles.columnReverse, inversionStyle] : horizontal ? [styles.row, inversionStyle] : inversionStyle;
+    cellStyle.push(styles.fullHeight);
 
     if (!CellRendererComponent) {
       return (
@@ -1601,6 +1602,9 @@ var styles = StyleSheet.create({
     left: 0,
     borderColor: 'red',
     borderWidth: 2
+  },
+  fullHeight: {
+    height: '100%',
   }
 });
 export default VirtualizedList;
\ No newline at end of file
